src/js/utils/animations.js

```diff
@@ -0,0 +1,144 @@
+import { css, keyframes } from 'styled-components';
```

Comment:
We should make sure StyledBox is also using this util functions of `animation` and it works as expected without backward compatibility issues.

Reply:
Removed all the helper functions in StyledBox.js that were already in utils and all of them still work as intended!